lib/run.rb

```diff
@@ -0,0 +1,14 @@
+require "mergometer"
```

Comment:
This does not seem to be in the right place.

I don't remember seeing any gem that places files directly in `lib/`.

This seems to be what the `bin/run` script should be.

Here's what I think we should have:

```rb
# exe/mergometer
#!/usr/bin/env ruby

require "mergometer/cli"

Mergometer::Cli.run
```

(See https://bundler.io/blog/2015/03/20/moving-bins-to-exe.html)

```rb
# lib/mergometer/cli.rb
require "mergometer"

module Mergometer
  class Cli
    def self.run
      new(ARGV).run
    end

    def initialize(argv)
      @argv = argv
    end

    def run
      # Here, the CLI's logic
    end

    private

      attr_reader :argv
  end
end
```

### Usage

```
gem install mergometer
OCTOKIT_ACCESS_TOKEN=<token> mergometer <report_name> <github_organization/repo_name>
```

Reply:
Sounds nice 👍

Reply:
Also how to make `mergometer` command work? does it automatically work using `bin/run`?

Reply:
@aadityataparia I tried and it seems to work as I documented it:
https://github.com/balvig/mergometer/pull/16

Reply:
https://github.com/balvig/mergometer/pull/16#issuecomment-397188921 🙏